app/vets-json-schema

```diff
@@ -1 +1 @@
-Subproject commit a819f86a70bed263ad01d49c3ff1691f13c15af2
+Subproject commit c4b69c3a5076d31e2d40d5ab7f053aab6bdbe938
```

Comment:
@mphprogrammer please remove this from your Gemfile. This is just because you you didn't do a `git submodule update`